### PR TITLE
Specialized in tag

### PR DESCRIPTION
**Pull request by THIRUSELVAN-V**
Specialized in tag finished

### CODE REVIEW

File: packages/ui/src/components/componentStyled/specializedInTag/index.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+'use client';
+export const SpecializedInTag = ({
+	icon,
```
Comment (premCrayon) [inline packages/ui/src/components/componentStyled/specializedInTag/index.tsx:7]: use interface

File: packages/ui/src/components/componentStyled/specializedInTag/index.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+	specializedIn,
+}: { icon?: React.ReactNode; specializedIn?: string }) => {
+	return (
+		<div className='w-[18.813rem] h-10 px-[0.625rem] flex items-center rounded-tl-lg rounded-bl-lg gap-1 font-[400] text-sm  bg-gradient-to-r from-[#F8971F1F] to-[#F8F8F83D]'>
```
Comment (premCrayon) [inline packages/ui/src/components/componentStyled/specializedInTag/index.tsx]: use from themes fonts,color,fontfamily

File: packages/ui/src/components/componentStyled/specializedInTag/index.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+			<div>{icon}</div>
```
Comment (premCrayon) [inline packages/ui/src/components/componentStyled/specializedInTag/index.tsx]: don't use div display only {icon}

File: packages/ui/src/components/componentStyled/specializedInTag/index.tsx [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+			<div>Specialized in</div>
```
Comment (premCrayon) [inline packages/ui/src/components/componentStyled/specializedInTag/index.tsx]: use p tag

File: packages/ui/src/components/componentStyled/specializedInTag/index.tsx [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+			<div className='font-[500]'>{specializedIn}</div>
```
Comment (premCrayon) [inline packages/ui/src/components/componentStyled/specializedInTag/index.tsx]: it also use p tag

Comment (THIRUSELVAN-V) [reply packages/ui/src/components/componentStyled/specializedInTag/index.tsx]: completed

File: packages/ui/src/components/componentStyled/specializedInTag/index.tsx (hunk 4 above)
Comment (THIRUSELVAN-V) [reply packages/ui/src/components/componentStyled/specializedInTag/index.tsx]: completed

File: packages/ui/src/components/componentStyled/specializedInTag/index.tsx (hunk 3 above)
Comment (THIRUSELVAN-V) [reply packages/ui/src/components/componentStyled/specializedInTag/index.tsx]: completed

File: packages/ui/src/components/componentStyled/specializedInTag/index.tsx (hunk 2 above)
Comment (THIRUSELVAN-V) [reply packages/ui/src/components/componentStyled/specializedInTag/index.tsx]: completed

File: packages/ui/src/components/componentStyled/specializedInTag/index.tsx (hunk 1 above)
Comment (THIRUSELVAN-V) [reply packages/ui/src/components/componentStyled/specializedInTag/index.tsx:7]: completed

**Pull request merged**